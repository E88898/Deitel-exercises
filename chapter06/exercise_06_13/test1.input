8.22
7.98
4.52
6.9999
3.345
