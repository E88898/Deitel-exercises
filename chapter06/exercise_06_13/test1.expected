Enter a number: 8.22 rounded is 8
Enter a number: 7.98 rounded is 8
Enter a number: 4.52 rounded is 5
Enter a number: 6.9999 rounded is 7
Enter a number: 3.345 rounded is 3
